pkgs/by-name/ba/babl/package.nix

```diff
@@ -1,53 +1,63 @@
 {
-  stdenv,
+  fetchFromGitLab,
+  git,
+  gi-docgen,
+  gobject-introspection,
+  lcms2,
   lib,
-  fetchurl,
+  librsvg,
   meson,
   ninja,
   pkg-config,
-  gi-docgen,
-  gobject-introspection,
-  lcms2,
+  stdenv,
   vala,
+  w3m,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "babl";
-  version = "0.1.110";
+  version = "0.1.112";
 
   outputs = [
     "out"
     "dev"
     "devdoc"
   ];
 
-  src = fetchurl {
-    url = "https://download.gimp.org/pub/babl/${lib.versions.majorMinor finalAttrs.version}/babl-${finalAttrs.version}.tar.xz";
-    hash = "sha256-v0e+dUDWJ1OJ9mQx7wMGTfU3YxXiQ9C6tEjGqnE/V0M=";
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "babl";
+    tag = "BABL_${builtins.replaceStrings [ "." ] [ "_" ] finalAttrs.version}";
+    hash = "sha256-Xlsecb8MB3FnIaXgbN1/l3FPqs8xxu6q7Qudo3fjSxI=";
   };
 
+  postUnpack = ''
+    cp source/git-version.h.in source/git-version.h
+    substituteInPlace source/git-version.h --replace-fail "@BABL_GIT_VERSION@" "${finalAttrs.src.tag}"
+  '';
+
   patches = [
     # Allow overriding path to dev output that will be hardcoded e.g. in pkg-config file.
     ./dev-prefix.patch
   ];
 
   nativeBuildInputs = [
+    git
+    gi-docgen
+    gobject-introspection
+    librsvg
```

Comment:
This is only used for stuff that is not installed so including it is pointless: https://github.com/GNOME/babl/blob/fd1d3105d1d58971a0c11c328afaa5e9a9831d51/docs/graphics/meson.build#L38